egs/librispeech/ASR/conformer_ctc/compute_ppl.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
```

Comment:
Can we have a new directory, e.g., `conformer_ctc_lm`, and leave `conformer_ctc` as it is.

Reply:
Would it use the model trained in the conformer_ctc dir for ASR decoding?  And how about the modeling code-- would that just be copied?  Soft linked?

Reply:
How about copying the modeling code?
If the modeling code is copied, I think the two can share model checkpoints if needed.
Or do you have other suggestions?

Reply:
I guess it's OK to copy.. seems a bit non-optimal to duplicate code, but I suppose it would keep the conformer_ctc code clean while we are still playing with this LM rescoring stuff, in case more changes are coming(?)

Reply:
RE where to put the scripts in the utils dir: we could possibly put them in a new directory ^/icefall/lm, but to do that we might want to look at them a bit more carefully to make sure they are well structured and well documented.  Would definitely need more documentation.  We can see what Fangjun thinks too.